config/stylelint.js

```diff
@@ -22,6 +22,133 @@ module.exports = {
     'selector-list-comma-newline-after': 'always-multi-line',
     'selector-no-id': true,
     'string-quotes': 'double',
-    'value-no-vendor-prefix': true
+    'value-no-vendor-prefix': true,
+    'declaration-block-properties-order': [
```

Comment:
Try to import sorting config and write smth like: `declaration-block-properties-order: config["postcss-sorting"]["sort-order"][3];`